docker-compose.test.yml

```diff
@@ -146,7 +182,8 @@ services:
     extends:
       file: docker-compose.yml
       service: discovery-provider
-    command: sleep infinity
+    entrypoint: sh -c '[ ! "$$1" = "test" ] && sleep inf || (shift; pytest "$$@")' -
```

Comment:
this seems more brittle to me because it depends on invoked args versus just letting the container come up as a testing container, so a debugger works, etc. why make this change?

Reply:
This should let the debugger work again